### PR TITLE
Renamed KernelHealth to DatabaseHealth

### PR DESCRIPTION
**Pull request by jimwebber**
Preparation for wider use of panic mechanism in CE.

### CODE REVIEW

File: enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java [hunk 1]
```diff
@@ -71,7 +71,7 @@
 
         LogRotation logRotation();
 
-        KernelHealth kernelHealth();
+        DatabaseHealth kernelHealth();
```
Comment (digitalstain) [inline enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java:74]: Need to rename the method as well

File: enterprise/com/src/test/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpackerTest.java [hunk 2]
```diff
@@ -268,10 +268,10 @@ public void shouldThrowInCaseOfFailureToAppend() throws Throwable
         }
     }
 
-    private KernelHealth newKernelHealth()
+    private DatabaseHealth newKernelHealth()
```
Comment (digitalstain) [inline enterprise/com/src/test/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpackerTest.java:271]: This method must be renamed to newDatabaseHealth()

File: community/kernel/src/test/java/org/neo4j/kernel/impl/transaction/DatabaseHealthTest.java [hunk 3]
```diff
@@ -59,17 +59,17 @@ public void shouldLogKernelPanicEvent() throws Exception
...
         String message = "Listen everybody... panic!";
         Exception exception = new Exception( message );
-        kernelHealth.panic( exception );
+        databaseHealth.panic( exception );
 
         // THEN
         logProvider.assertAtLeastOnce(
-                inLog( KernelHealth.class ).error(
+                inLog( DatabaseHealth.class ).error(
                         is("setting TM not OK. Kernel has encountered some problem, please perform necessary action (tx recovery/restart)" ),
```
Comment (digitalstain) [inline community/kernel/src/test/java/org/neo4j/kernel/impl/transaction/DatabaseHealthTest.java:73]: This is causing the failing test. Interestingly, although the logged string is public in the class, it is not used here. We should fix this.

**Comment by digitalstain** [comment]
@jimwebber Needs rebase. Already.

**Comment by jimwebber** [comment]
@digitalstain will do.

**Comment by jimwebber** [comment]
The rebase for this is a bit of work compared to the original work. I'm gonna close this one down and re-do the work on head.

**Pull request closed without merging**